other-examples/aws/memcached/nodejs/src/public/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
It should be possible to demonstrate a relationship to memcached without a UI. The code for the UI bloats this example and makes it difficult to hone in on what you're attempting to demonstrate.